documentation/source/user_section/tutorials.rst

```diff
@@ -37,6 +44,12 @@ We provide the following hands-on tutorials for SCT's command-line tools.
 
 #. :doc:`Analysis pipelines with SCT <tutorials/analysis-pipelines-with-sct>`
 
+Others sources of knowledge
+***************************
+
+We provide a **free** in-person SCT course each year. You can know more about these courses in the :ref:`course section<courses>`.
```

Comment:
Slight wording tweak:

```suggestion
SCT Course
**********

If you would prefer to learn how to use SCT in a guided workshop setting, we provide an in-person SCT course each year. You can learn more about past and future courses in the :ref:`course section<courses>`.
```

(I added an introductory sentence to help it flow a little better, and also removed the mention of **free** as I believe our most recent SCT Course at the Martinos Center involved a fee?)